upup/pkg/fi/cloudup/networking.go

```diff
@@ -36,88 +36,127 @@ import (
 
 const (
 	// defaultCNIAssetAmd64K8s_11 is the CNI tarball for k8s >= 1.11
-	defaultCNIAssetAmd64K8s_11              = "https://storage.googleapis.com/kubernetes-release/network-plugins/cni-plugins-amd64-v0.7.5.tgz"
-	defaultCNIAssetAmd64SHA256StringK8s1_11 = "3ca15c0a18ee830520cf3a95408be826cbd255a1535a38e0be9608b25ad8bf64"
```

Comment:
why were the default cni asset SHAs removed?

Reply:
The SHAs were added because initially the tarballs didn't have hash files in the download repo. This was addressed some time ago:
https://github.com/kubernetes/kops/blob/781279e268847579b47a87600380675317233b58/pkg/assets/builder.go#L249

Reply:
Ah I see
```
gsutil ls gs://kubernetes-release/network-plugins/
gs://kubernetes-release/network-plugins/cni-plugins-amd64-v0.7.5.tgz
gs://kubernetes-release/network-plugins/cni-plugins-amd64-v0.7.5.tgz.asc
gs://kubernetes-release/network-plugins/cni-plugins-amd64-v0.7.5.tgz.sha1
gs://kubernetes-release/network-plugins/cni-plugins-amd64-v0.7.5.tgz.sha256
gs://kubernetes-release/network-plugins/cni-plugins-amd64-v0.7.5.tgz.sha512
gs://kubernetes-release/network-plugins/cni-plugins-arm64-v0.7.5.tgz
gs://kubernetes-release/network-plugins/cni-plugins-arm64-v0.7.5.tgz.asc
gs://kubernetes-release/network-plugins/cni-plugins-arm64-v0.7.5.tgz.sha1
gs://kubernetes-release/network-plugins/cni-plugins-arm64-v0.7.5.tgz.sha256
gs://kubernetes-release/network-plugins/cni-plugins-arm64-v0.7.5.tgz.sha512
```

```
gsutil ls gs://k8s-artifacts-cni/release/v0.8.6/
gs://k8s-artifacts-cni/release/v0.8.6/cni-plugins-linux-amd64-v0.8.6.tgz
gs://k8s-artifacts-cni/release/v0.8.6/cni-plugins-linux-amd64-v0.8.6.tgz.asc
gs://k8s-artifacts-cni/release/v0.8.6/cni-plugins-linux-amd64-v0.8.6.tgz.sha1
gs://k8s-artifacts-cni/release/v0.8.6/cni-plugins-linux-amd64-v0.8.6.tgz.sha256
gs://k8s-artifacts-cni/release/v0.8.6/cni-plugins-linux-amd64-v0.8.6.tgz.sha512
gs://k8s-artifacts-cni/release/v0.8.6/cni-plugins-linux-arm64-v0.8.6.tgz
gs://k8s-artifacts-cni/release/v0.8.6/cni-plugins-linux-arm64-v0.8.6.tgz.asc
gs://k8s-artifacts-cni/release/v0.8.6/cni-plugins-linux-arm64-v0.8.6.tgz.sha1
gs://k8s-artifacts-cni/release/v0.8.6/cni-plugins-linux-arm64-v0.8.6.tgz.sha256
gs://k8s-artifacts-cni/release/v0.8.6/cni-plugins-linux-arm64-v0.8.6.tgz.sha512
```